include/uapi/sound/sof/ext_manifest.h

```diff
@@ -51,11 +51,24 @@ struct sof_ext_man_header {
 
 /* Now define extended manifest elements */
 
+/* Extended manifest elements identificators */
```

Comment:
Can we please fix the prefix in the commit title for this one and the remaining ones to ASoC: SOF: XMan -> ASoC: SOF: ext_manifest? Xman is just not intuitive

Reply:
changed